saleor/app/management/commands/create_app.py

```diff
@@ -20,6 +21,12 @@ class Command(BaseCommand):
 
     def add_arguments(self, parser: CommandParser) -> None:
         parser.add_argument("name", type=str)
+        parser.add_argument(
+            "--identifier",
+            dest="identifier",
+            default="",
+            help="Canonical app ID.",
```

Comment:
I would add here: "If not provided, the identifier will be generated based on app.id"